package.json

```diff
@@ -17,10 +17,11 @@
     "build-js": "node scripts/build.js",
     "build:lint:fix": "npm run build:lint -- --fix",
     "build:lint": "eslint 'src/**' --ext .js,.jsx --env browser,node",
-    "build": "npm run build:index && rm -rf lib && NODE_ENV=production babel src --out-dir lib --ignore \"src/**/*.spec.js\",\"src/**/*.test.js\",\"src/**/*.Component.js\",\"src/_playground/*\"",
+    "build": "npm run build:index && rm -rf lib && NODE_ENV=production babel src --out-dir lib --ignore \"src/**/*.spec.js\",\"src/**/*.test.js\",\"src/**/*.Component.js\",\"src/_playground/*\" && npm run copyfiles",
     "build:index": "babel-node devtools/buildIndexFiles.js",
     "config:lint": "eslint 'config/**' --ext .js,.jsx --env browser,node",
     "config:lint:fix": "npm run config:lint -- --fix",
+    "copyfiles": "cp -rf package.json README.md NOTICE.txt LICENSE.txt CHANGELOG.md lib",
```

Comment:
Can we change this to `build:copyFiles` or something like that so we know it has to do with building the lib?